recipes-core/system-api/system-api_v0.7.0.bb

```diff
@@ -10,8 +10,10 @@ INITSCRIPT_PARAMS = "defaults 81"
 
 GO_IMPORT = "github.com/flashbots/system-api"
 SRC_URI = "git://${GO_IMPORT};protocol=https;branch=main \
-           file://system-api-init"
-SRCREV = "v0.4.0"
+           file://system-api-init \
+           file://basic-auth-secret.mustache \
```

Comment:
what is this for, and why does the secret hash need a mustache?

Reply:
removed